src/main/java/com/suse/saltstack/netapi/client/SaltStackClient.java

```diff
@@ -385,4 +386,19 @@ public Job call() throws SaltStackException {
         };
         return executor.submit(callable);
     }
+
+  public Stats stats() throws SaltStackException {
```

Comment:
I also agree with @lucidd and @moio as we currently have two versions of every method.